### ISSUE TITLE
Add `RMSE` metric

### ISSUE THREAD

**Issue by Mr-Geekman** [tinkoff-ai/etna#1031]
### 🚀 Feature Request

RMSE has more interpretability.

### Proposal

1. Add class `RMSE` into `etna/metrics/metrics.py`
2. It should be like `MSE` but use parameter `squared=False`, look at [documentation](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html).
3. Add tests like for `MSE`.

### Test cases

1. Make similar tests like for `MSE` and other metrics.
